### PR TITLE
Added implementation of fenwick tree

### PR DESCRIPTION
**Pull request by ShimoniSinha2019H1030019G**
I have created an implementation that constructs fenwick tree and shows sum and update queries on the tree created.

### CODE REVIEW

**Comment by ShimoniSinha2019H1030019G** [comment]
Can someone please review and comment/merge the pull request.

File: algorithms/tree/fenwick_tree.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    # Create and initialize bit_ree[] as 0 
+    bit_tree = [0]*(n+1) 
+  
+    # Store the actual values in bit_ree[] using update() 
+    for i in range(n): 
+        update_bit(bit_tree, n, i, arr[i]) 
+
+    return bit_tree 
+
+if __name__ == "__main__":
```
Comment (ericklarac) [inline algorithms/tree/fenwick_tree.py:74]: This test case should be moved to the tests file.

File: algorithms/tree/fenwick_tree.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+There are two solutions that can perform both the query and update operations in O(logn) time.
+1. Fenwick Tree
+2. Segment Tree
+
+Compared with Segment Tree, Binary Indexed Tree requires less space and is easier to implement.
+"""
+
+def get_sum(bit_tree, i):
```
Comment (ericklarac) [inline algorithms/tree/fenwick_tree.py:22]: Conver this into a class, to use it as a data structure.

**Review by ericklarac — CHANGES_REQUESTED** [review]

File: algorithms/tree/fenwick_tree.py (hunk 1 above)
Comment (ShimoniSinha2019H1030019G) [reply algorithms/tree/fenwick_tree.py]: Updated in the new commit.

File: algorithms/tree/fenwick_tree.py (hunk 2 above)
Comment (ShimoniSinha2019H1030019G) [reply algorithms/tree/fenwick_tree.py]: Updated in the new commit

**Comment by ShimoniSinha2019H1030019G** [comment]
@ericklarac  I have updated the changes requested.

**Review by ericklarac — APPROVED** [review]

**Pull request merged**